inference.py

```diff
@@ -282,6 +281,13 @@ def inference_multimer_model(args):
 
     out = result_q.get()
 
+
+    # Save the prediction result .pkl
+    prediction_result_path = os.path.join(args.output_dir,
+        f'{output_prefix}_{args.model_name}_.pkl')
+    with open(prediction_result_path, 'wb') as f:
+        pickle.dump(out, f, protocol=4)
```

Comment:
Why not just keep the protocol default?

Reply:
I agree, I have now modified it to default.